.github/workflows/maven.yml

```diff
@@ -21,5 +25,30 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '17'
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Build Backend
-        run: cd power-pay-backend && mvn package
\ No newline at end of file
+        run: cd power-pay-backend && mvn package
+
+      - id: string
+        uses: ASzc/change-string-case-action@v6
+        with:
+          string: XyZzY
+      - id: Output_Lowercase_String
+        run: echo ${{ steps.string.outputs.lowercase }}
```

Comment:
What is the usage?

Reply:
To display the converted value of the repository

Reply:
Yes, but what for? Ok, you displayed it, Now, what next?

Reply:
The problem is still there

Reply:
@stephane-segning The string was modified to the github.repository

Reply:
Yes, but you're not using it. You're just echo-ing it. Why not use it at the place which is causing the issue?